news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/susko3.jpg)](https://osu.ppy.sh/users/18945305)
+
+Tirelessly striving to fix bugs and bridge the differences between computing platforms, Susko3’s efforts in lazer development have proven instrumental in pushing the project forward in recent times. Their expertise in both lazer’s underlying development framework and other libraries (such as SDL) has proven integral to ongoing projects, and we are tremendously thankful for their dedication and support.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/stanr.jpg)](https://osu.ppy.sh/users/7217455)
+
+A mainstay presence in the performance points development community, StanR has been at the forefront of making meaningful changes to osu!'s key performance algorithms for a considerable length of time at this point, contributing new changes of his own and vetting those made by others. While this may not sound like much at first glance, his contributions represent hundreds of hours of work and have manifested in scoring becoming fairer and more fun for everyone.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/apo11o.jpg)](https://osu.ppy.sh/users/9558549)
+
+While some Beatmap Nominators mod only maps, Apo11o has gone a step beyond and taken it upon himself to mod the scoring algorithm as well in both major versions of osu!. The level of mathematical wizardry and game knowledge this requires cannot be understated, and to maintain an active presence in the pp development scene while meeting the monthly rigours of BN life on top of it all belies an utterly extraordinary dedication to making osu! more awesome.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/mbmasher.jpg)](https://osu.ppy.sh/users/4498616)
+
+Taking it upon himself to bring light into the (quite literal) darkness, MBmasher turned his attention upon the scoring particulars of the Flashlight mod and found them terribly lacking. So, he boldly charted ground none before him had dared to tread, and wrote an almost entirely new algorithm from the bottom up. Before him, Flashlight was a funny novelty mod with a fairly static score multiplier, and after him, it is appropriately acknowledged in code as the nuanced and difficult playstyle that it truly is.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/walavouchey.jpg)](https://osu.ppy.sh/users/5773079)
+
+Ever present in the osu-wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.
```

Comment:
"osu-wiki" and (later) "osu!wiki", similar to my comment about lazer wording. I think all should be changed to "osu! wiki" as that's the proper:tm: name used on the wiki itself